.nsprc

```diff
@@ -0,0 +1,4 @@
+{
+  // https://github.com/nodesecurity/nsp#exceptions
+  "exceptions": []
+}
```

Comment:
Technically not really "used" at the moment, but just some pre-emptive scaffolding for when there is a vulnerability in one of the nested dependencies and you can't fix it immediately but you also don't want your build system perma-failing until a un-vuln module is released.